modules/jlink/artifacts/opt/jboss/container/java/jlink/mkjreimage.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# TODO: Still Needed?
+set -euo pipefail
+
+outdir="${outdir-spring-boot-jre}"
+depsfile="module-deps.txt"
+
+test -f $depsfile
```

Comment:
As of 4adca3fd57649e468bb81e3c6a1a5cb8026a25a6, s2i build fails. Log excerpt:

```log
...
I1025 13:56:05.972669  215353 sti.go:715] + source /opt/jboss/container/java/jlink/mkjreimage.sh
I1025 13:56:05.972709  215353 sti.go:715] ++ set -euo pipefail
I1025 13:56:05.972737  215353 sti.go:715] ++ outdir=spring-boot-jre
I1025 13:56:05.972745  215353 sti.go:715] ++ depsfile=module-deps.txt
I1025 13:56:05.972759  215353 sti.go:715] ++ test -f module-deps.txt
...
```

The output file name of the previous script is `deps.txt`. However ,since that's wrapped in `generate_deps`, that code isn't actually executed at the point its file is sourced; the code in `mkjreimage` is not yet wrapped, so it does; so, no deps files (of any file name) exist yet, but please also check the filenames and the ordering.